document/js-api/index.bs

```diff
@@ -786,7 +786,7 @@ Note: Exported Functions do not have a \[[Construct]] method and thus it is not
 The algorithm <dfn>ToJSValue</dfn>(|w|) coerces a [=WebAssembly value=] to a JavaScript value performs the following steps:
 
 Assert: |w| is not of the form [=𝗂𝟨𝟦.𝖼𝗈𝗇𝗌𝗍=] |i64|.
-1. If |w| is of the form [=𝗂𝟥𝟤.𝖼𝗈𝗇𝗌𝗍=] |i32|, return [=the Number value=] for |i32|.
+1. If |w| is of the form [=𝗂𝟥𝟤.𝖼𝗈𝗇𝗌𝗍=] |i32|, return [=the Number value=] for |i32| when interpreted as a signed integer.
```

Comment:
How about invoking [signed_32](http://webassembly.github.io/spec/core/exec/numerics.html#aux-signed)(i32) explicitly, to make precise what that means?

Reply:
That function is perfect for this; thanks for the cross-reference. Switched to use it.